src/main/java/bob/Parser.java

```diff
@@ -0,0 +1,153 @@
+package bob;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents Parser object, to read and respond to user input
+ */
+public class Parser {
+
+    private boolean toExit = false;
+
+    /**
+     * Parses through user input and determines response
+     *
+     * @param reply user input
+     * @param tasks current list of tasks
+     * @param storage storage to save and read files
+     * @param ui ui to determine text responses to user
+     */
+    //referenced https://github.com/Donovan9617/ip/blob/master/src/main/java/Duke/Parser.java for structure
+    public void parse(String reply, TaskList tasks, Storage storage, Ui ui) {
+        String[] splitReply = reply.split(" ");
+        String command = splitReply[0].toLowerCase();
+        switch(command) {
+        case "bye":
+            ui.printLine();
+            ui.sayGoodbye();
+            ui.printLine();
+            this.toExit = true;
+            break;
+        case "lift":
```

Comment:
Should this be "list" instead of "lift"?
```suggestion
        case "list":
```

Reply:
Yes, almost missed that, thank you!